app/src/main/java/io/gnosis/safe/ui/transactions/TransactionListFragment.kt

```diff
@@ -53,8 +53,15 @@ class TransactionListFragment : SafeOverviewBaseFragment<FragmentTransactionList
 
         adapter.addLoadStateListener { loadState ->
             if (lifecycle.currentState.isAtLeast(Lifecycle.State.STARTED)) {
-                binding.transactions.isVisible = loadState.refresh is LoadState.NotLoading
                 binding.progress.isVisible = loadState.refresh is LoadState.Loading
+                val append = loadState.append
+                if(append is LoadState.Error) {
+                    handleError(append.error)
+                }
+                val prepend = loadState.prepend
+                if(prepend is LoadState.Error) {
+                    handleError(prepend.error)
+                }
             }
         }
         adapter.addDataRefreshListener { isEmpty ->
```

Comment:
Is this method deprecated? https://developer.android.com/reference/kotlin/androidx/paging/PagingDataAdapter#adddatarefreshlistener